apps/services/university-gateway/src/app/modules/program/test/program.service.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Type } from '@nestjs/common'
+import { getConnectionToken } from '@nestjs/sequelize'
+import assert from 'assert'
+import { Sequelize } from 'sequelize-typescript'
+import request, { SuperTest, Test } from 'supertest'
+
+import { createCurrentUser } from '@island.is/testing/fixtures'
+import { TestApp, setupApp, truncate } from '@island.is/testing/nest'
+
+import { FixtureFactory } from '../../../../../test/fixtureFactory'
+import { SequelizeConfigService } from '../../../../sequelizeConfig.service'
+import { AppModule } from '../../../app.module'
+// import { ProgramService } from '../program.service'
```

Comment:
Why is this commented out, is this something that'll be commented back in at a later point?